Sources/GeneralIOs/Commands/Bootstrap.swift

```diff
@@ -44,6 +47,10 @@ final class Bootstrap: ParsableCommand {
             static let firebase: String = "firebase"
             static let swiftgen: String = "swiftgen"
             static let licenseplist: String = "licenseplist"
+            static let shell: String = "shell"
+            static let commandName: String = "name"
```

Comment:
Why name constant commandName, and are not simple name?

Reply:
simple name is too common naming for the variable that have very limited usage. I prefer to call it more specific like `shellCommandName` then simple `name`